implementations/code_df/code_changes_git.py

```diff
@@ -110,3 +111,12 @@ def __str__(self):
     # time-series on a monthly basis considering only master commits
     print("The number of commits created on a monthly basis is:")
     print(changes.time_series(period='M'))
+
+    # considering commits only that contain either [api] or [backend]
+    # into their message
+    tags = ["[api]", "[backend]"]
+    for tag in tags:
+        print("Code_Changes, only for the commits that contain {} into their message: {}".format(
+            tag,
+            CodeChangesGit(items, conds=[CommitByTag(tag)]).compute()
```

Comment:
A suggestion: instead of having to use a for loop and creating multiple objects, maybe you could pass the entire tags list to `CommitByTag` so that it includes those commits which satisfy any one of the tags provided in the list, and modify its implementation in `conditions.py` accordingly.

Let me know if this is not clear, and I'll elaborate.

Reply:
It's clear, I'll change this.
Thanks!

Reply:
After implementing and testing your suggestion, I think it's a good idea if we want to count the commits whose message starts with one of tags, but if we want to compare the number of commits for each tag, then we don't do this without using the for loop.

Reply:
You're right when you say that. But the goal of the classes in `conditions.py` is to restrict our analysis. Anytime someone wants to compare the relationship between commits with different tags, they can just not pass the `CommitByTag` condition and then perform the analysis on all commits, segregated by tag using a loop if they prefer.

Later on, if we think a cleaner way to do this is required, we could just add segregating methods for each of the conditions mentioned in `conditions.py`. For now, maybe lets just leave it at that for the sake of consistency.